extensions/blocks/amazon/index.js

```diff
@@ -19,44 +18,19 @@ import './editor.scss';
 export const name = 'amazon';
 export const title = __( 'Amazon', 'jetpack' );
 export const settings = {
+	attributes,
 	title,
-	description: (
-		<Fragment>
-			<p>{ __( 'Amazon', 'jetpack' ) }</p>
-			<ExternalLink href="#">{ __( 'Learn more about Amazon', 'jetpack' ) }</ExternalLink>
-		</Fragment>
-	),
-	/* @TODO Add the icon. You can use one of these https://material.io/tools/icons/?style=outline */
-	icon: renderMaterialIcon(
-		<Path d="M9 15h2V9H9v6zm1-10c-.5 0-1 .5-1 1s.5 1 1 1 1-.5 1-1-.5-1-1-1zm0-4c-5 0-9 4-9 9s4 9 9 9 9-4 9-9-4-9-9-9zm0 16c-3.9 0-7-3.1-7-7s3.1-7 7-7 7 3.1 7 7-3.1 7-7 7z" />
-	),
+	description: __( 'Promote Amazon products and earn a commission from sales.', 'jetpack' ),
+	icon,
 	category: 'jetpack',
-	keywords: [],
+	keywords: [ __( 'amazon', 'jetpack' ), __( 'affiliate', 'jetpack' ) ],
```

Comment:
We probably don't need to add the `amazon` keyword here, since you'll already get a result for that keyword because of the block name.

Reply:
I agree, I'm still waiting for better ideas: https://wp.me/pb5gDS-h1#comment-718